trace/cloud-trace-demo-app/app/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM python:3.8.0
+
+RUN pip3 install --upgrade pip
+
+RUN apt-get update \
+ && apt-get clean \
+ && rm -rf /var/lib/apt/lists/*
+
+COPY requirements.txt /app/
+
+WORKDIR /app
+RUN pip3 install -r requirements.txt
+RUN pip3 install opencensus-ext-stackdriver
+RUN pip3 install grpcio
+COPY *.py /app/
+
+ENTRYPOINT ["python"]
+CMD ["./app.py"]
```

Comment:
The default flask server isn't really suitable for running production code, we should encourage users to use something like `gunicorn` - you can find an example of how to do this [here](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/09bc50d610741d693c4e99e03854822b37a647ba/cloud-sql/mysql/sqlalchemy/Dockerfile#L37).

Reply:
Done.